src/directives/formly-field.js

```diff
@@ -59,18 +59,25 @@ function formlyField($http, $q, $compile, $templateCache, $interpolate, formlyCo
 
     // function definitions
     function runExpressions() {
+      const deferred = $q.defer()
```

Comment:
I did have the same thought but changing it to return the $q.all from inside the $timeout made it not work. I'm not sure why because I would've assumed they'd be equivalent.